core/src/main/csharp/ch/cyberduck/core/CredentialManagerPasswordStore.cs

```diff
@@ -61,9 +64,12 @@ public override void delete(Host bookmark)
                 logger.info(string.Format("Delete password for bookmark {0}", bookmark));
             }
             var target = ToUri(bookmark);
-            if (!WinCredentialManager.RemoveCredentials(target.AbsoluteUri))
+            foreach (Uri descriptor in target)
             {
-                base.delete(bookmark);
+                if (!WinCredentialManager.RemoveCredentials(descriptor.AbsoluteUri))
+                {
+                    base.delete(bookmark);
```

Comment:
This will get called multiple times.
Unconditionally call base.delete after loop then.